From 198a441033077c5e4f495fec2d2d24f2b24fcb0c Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 27 Feb 2020 16:17:09 +0000
Subject: [PATCH 1005/1077] MANTIS 0056312 Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses add IdDocument.xsd

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1555 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/idl/Cuss/IdDocument.xsd            | 77 +++++++++++++++++++
 1 file changed, 77 insertions(+)
 create mode 100644 SelfServiceCommon/Massai/idl/Cuss/IdDocument.xsd

diff --git a/SelfServiceCommon/Massai/idl/Cuss/IdDocument.xsd b/SelfServiceCommon/Massai/idl/Cuss/IdDocument.xsd
new file mode 100644
index 00000000..fbcb9d72
--- /dev/null
+++ b/SelfServiceCommon/Massai/idl/Cuss/IdDocument.xsd
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Mit XMLSpy v2020 (x64) (http://www.altova.com) von Materna Information & Communications SE (Materna Information & Communications SE) bearbeitet -->
+<xs:schema xmlns:iddoc="http://www.materna.com/CUSS/IdDocument-1.0" xmlns:oneIdTypes="http://www.materna.com/CUSS/OneIdTypes-1.0" xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.materna.com/CUSS/IdDocument-1.0" elementFormDefault="qualified" attributeFormDefault="unqualified">
+
+	<xs:import namespace="http://www.materna.com/CUSS/OneIdTypes-1.0" schemaLocation="OneIdTypes.xsd"/>
+
+	<xs:complexType name="version">
+		<xs:annotation>
+			<xs:documentation>schema version; increase fixed values of version when schema is changed</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="majorVersion" type="xs:int" fixed="1">
+				<xs:annotation>
+					<xs:documentation>major version</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="minorVersion" type="xs:int" fixed="0">
+				<xs:annotation>
+					<xs:documentation>minor version</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="revision" type="xs:int" fixed="0">
+				<xs:annotation>
+					<xs:documentation>revision</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+
+	<xs:element name="data-item">
+		<xs:annotation>
+			<xs:documentation>General purpose biometric function request and/or response individual data item.</xs:documentation>
+		</xs:annotation>
+		<xs:complexType>
+			<xs:simpleContent>
+				<xs:extension base="xs:string">
+					<xs:attribute name="data-item-name" type="xs:string" use="required"/>
+				</xs:extension>
+			</xs:simpleContent>
+		</xs:complexType>
+	</xs:element>
+	<xs:element name="data-list">
+		<xs:annotation>
+			<xs:documentation>General purpose biometric function request and/or response data item list.</xs:documentation>
+		</xs:annotation>
+		<xs:complexType>
+			<xs:sequence>
+				<xs:element ref="iddoc:data-item" maxOccurs="unbounded"/>
+			</xs:sequence>
+		</xs:complexType>
+	</xs:element>
+
+	<xs:element name="IdDocument">
+		<xs:annotation>
+			<xs:documentation>OneID Application Data for the CUSS Application Transfer from the OneID Application to SBD/CKI or vice versa.</xs:documentation>
+		</xs:annotation>
+		<xs:complexType>
+			<xs:sequence>
+				<xs:element name="interfaceVersion" type="iddoc:version">
+					<xs:annotation>
+						<xs:documentation>Version of the OneID Application Data  for the CUSS Application Transfer (i.e. XML Schema version).</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element name="idDocument" type="oneIdTypes:idDocumentData">
+					<xs:annotation>
+						<xs:documentation>Structured id document data (incl. optional MRZ tracks and document image)</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+				<xs:element ref="iddoc:data-list" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation>Optional key value pairs in case something is missing ;-)</xs:documentation>
+					</xs:annotation>
+				</xs:element>
+			</xs:sequence>
+		</xs:complexType>
+	</xs:element>
+</xs:schema>
-- 
2.41.0.windows.1

